src/support_degree.c

```diff
@@ -0,0 +1,23 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<math.h>
+int main()
+{
+double x[4] = {2.0,4.0,3.75,3.55};
+double **D;
```

Comment:
Pointer must be prefix with p ;followed by variable name

Reply:
But that's not a coding convention specified for the project, we are just sticking with those in CULearn.